src/components/chart/state-history-chart-timeline.ts

```diff
@@ -214,7 +214,7 @@ export class StateHistoryChartTimeline extends LitElement {
         },
         axisLabel: {
           show: showNames,
-          width: labelWidth,
```

Comment:
I am guessing letter-spacing will fix this as well but can't repro it so pls check. It also might need to be set to something other than `normal` for this

Reply:
I still have the issue with `labelWidth - labelMargin`.
No issue with `labelWidth` only.

Reply:
alright, removed that part. Will tackle it separately